web/src/components/shared/storageDefinitions.ts

```diff
@@ -14,127 +15,149 @@ export const storageDefinitions: StorageDefinition[] = [
     value: "data_safe_haven_file_transfer",
     definition:
       "If sending files via the Data Safe Haven File Transfer Portal to an email recipient, users must use a different password delivery mechanism to the email address used.",
+    riskScore: 1,
   },
   {
     name: "Desktop@UCL staff computer",
     value: "desktop_ucl_staff",
     definition:
       "UCL Information Services Division will issue on request a standard desktop installation called Desktop@UCL which includes encryption of the device's storage and anti-malware. However, some departments have not migrated to the Desktop@UCL service and will issue devices which are not necessarily as secure by default.",
+    riskScore: 2,
   },
   {
     name: "DfE network",
     value: "dfe_network",
     definition:
       "The Department for Education's Data Sharing Agreement makes these data highly confidential by default. The contract must be adhered to. Advice is best sought from IG Advisory (slms.pid@ucl.ac.uk) before entering these agreements.",
+    riskScore: 3,
   },
   {
     name: "Email",
     value: "email",
     definition:
       "Consult with IG Advisory (slms.pid@ucl.ac.uk) if you need to use email to send or receive research participants' personal data.",
+    riskScore: 5,
   },
   {
     name: "Fax",
     value: "fax",
     definition:
       "Consult with IG Advisory (slms.pid@ucl.ac.uk) if you need to use fax to send or receive research participants' personal data.",
+    riskScore: 5,
   },
   {
     name: "GalaxKey",
     value: "galaxkey",
     definition: "No further requirements.",
+    riskScore: 3,
   },
   {
     name: "Handheld recording equipment (secured)",
     value: "handheld_recording_secured",
     definition:
       "Secure recording devices must either: (a) have device or file encryption requiring a strong, randomly generated password, or (b) be used only in the following way: - where the files can be securely transferred to the Data Safe Haven or an equally secure platform - the device files erased immediately upon confirmation of the completed transfer - without having to travel between sites",
+    riskScore: 2,
   },
   {
     name: "Handheld recording equipment (unsecured)",
     value: "handheld_recording_unsecured",
     definition:
       "Consult with IG Advisory (infogov@ucl.ac.uk) if you need to use unsecured devices to store research participants' personal data.",
+    riskScore: 5,
   },
   {
     name: "Internet download/upload (secured)",
     value: "internet_download_upload_secured",
     definition: "Authentication required to access, session secured with end to end encryption (HTTPS or VPN)",
+    riskScore: 3,
   },
   {
     name: "Internet download/upload (unsecured)",
     value: "internet_download_upload_unsecured",
     definition:
       "Consult with IG Advisory (slms.pid@ucl.ac.uk) if you need to download or upload research participants' personal data without a secure connection.",
+    riskScore: 3,
```

Comment:
strongly disagree with this – must be higher than secured internet connection
```suggestion
    riskScore: 5,
```

Reply:
fair